.github/workflows/release.yml

```diff
@@ -44,6 +44,17 @@ jobs:
 
       - name: Run storybook build
         run: make storybook
+
+    - name: Commit storybook changes
+      uses: EndBug/add-and-commit@v4
+      with:
+        author_name: Max Strübing
+        author_email: max.struebing@farbenmeer.de
+        message: "docs: Update storybook"
+        add: "docs"
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Indention is not correct

Reply:
Readme says it is correct: https://github.com/EndBug/add-and-commit/blob/master/README.md

Reply:
No its not, you have to add 2 spaces on each line, 
```yml
    - name: Commit storybook changes
```
is not on the same line with 
```yml
      - name: Run storybook build
```

See here:
![Bildschirmfoto 2020-06-23 um 18 57 55](https://user-images.githubusercontent.com/14888529/85432723-a746ce00-b583-11ea-9bca-d8e7b208c4f3.png)

Reply:
![Screenshot_2020-06-24_09-40-39](https://user-images.githubusercontent.com/12071529/85517129-42c95480-b5ff-11ea-89f8-722b0d6554e8.png)

Reply:
Nevermind, I will change it and we will see what happens :)

Reply:
Ah, sorry, I was totally looking on the wrong line, my bad.